traitsui/ui_editors/data_frame_editor.py

```diff
@@ -395,3 +401,19 @@ def _get_klass(self):
         """ The class used to construct editor objects.
         """
         return toolkit_object("data_frame_editor:_DataFrameEditor")
+    
+    def _get_scroll_to_row_hint(self):
+        warnings.warn(
```

Comment:
"warnings" is undefined at this point, so accesses to `scroll_to_row_hint` will raise `NameError` rather than warn.

Reply:
A test would have caught this. Just saying.

Reply:
😅
I've opened #1753